nautilus_trader/model/objects.pyx

```diff
@@ -67,8 +73,10 @@ PRICE_MIN = RUST_PRICE_MIN
 MONEY_MAX = RUST_MONEY_MAX
 MONEY_MIN = RUST_MONEY_MIN
 
-FIXED_PRECISION = RUST_FIXED_PRECISION
-FIXED_SCALAR = RUST_FIXED_SCALAR
+HIGH_PRECISION = PRECISION == 16
```

Comment:
It'll be a safer option to load this from the cdef HIGH_PRECISION value defined in model.pxd. Just in case the digits of precision change for high precision it'll very difficult to track down why this is failing.

Reply:
Also had a similar thought, had to add another constant generated at compile time, as `HIGH_PRECISION` was already occupied with the DEF, but now achieving what we were after:

```
HIGH_PRECISION = RUST_HIGH_PRECISION_MODE
```